returnn/datasets/multi_proc.py

```diff
@@ -33,13 +33,18 @@ def __init__(
         dataset: Dict[str, Any],
         num_workers: int,
         buffer_size: int,
+        sharding_method: str = "seq_order",
         _meta_info_cache: Optional[Dict[str, Any]] = None,
         **kwargs,
     ):
         """
         :param dataset: the dataset to use
         :param num_workers: number of workers to use
         :param buffer_size: buffer size for each worker, amount of seqs to prefetch
+        :param sharding_method: which method to use for sharding the data across the worker procs.
+            Default is ``seq_order``, which fetches the full list of seq indices from worker 0, and then
+            distributes shards of that to the other workers.
```

Comment:
```suggestion
            Default is ``seq_order``, which fetches the full list of seq indices,
            and then distributes shards of that to the other workers.
```
"from worker 0" is an implementation detail, and could also change (has already been changed in the past), and not relevant/important.